app/helpers/hera_helper.rb

```diff
@@ -68,6 +71,19 @@ def navbar_brand
     end
   end
 
+  def node_types_collection
```

Comment:
This looks like it should be in a node-specific helper defined in a the controller (or node helper) and not in the more general hera helper

Reply:
Agreed, but alas, this is where the layout helpers are. There's still some legacy problems around the rendering of the Nodes tree, which isn't handled nicely by a NodesController, so again, a battle for another day.